### DISCUSSION TITLE
Minor issue

### DISCUSSION THREAD

**Discussion by iocoker** [fabricjs/fabric.js#8553] [General]
Not sure if this requires a pull request, it's a minor bug. V6 doesn't compile for me due to a `const` declaration here:
https://github.com/fabricjs/fabric.js/blob/695cf10369b8b2815c607f0c2f79edb0bea91a36/src/shapes/group.class.ts#L684

Which is later re-assigned few lines down here:
https://github.com/fabricjs/fabric.js/blob/695cf10369b8b2815c607f0c2f79edb0bea91a36/src/shapes/group.class.ts#L688

**Comment by ShaMan123** [comment]
@iocoker this is a severe bug.
I agree, it is very minor but will cause fabric to break.
It wasn't migrated properly to TS.

**Reply by iocoker** [reply]
Sure, just did.

**Comment by ShaMan123** [comment]
I created an issue from this thread.
Please don't hesitate and open issues and PR.
This is a **hell yes** in terms of motivation.
